src/main.js

```diff
@@ -135,9 +135,6 @@ var argv = yargs.usage('Usage: diff2html [options] -- [diff args]')
   .example('diff2html -i file -- my-file-diff.diff', 'reading the input from a file')
   .example('diff2html -f json -o stdout -- -M HEAD~1', 'print json format to stdout')
   .example('diff2html -F my-pretty-diff.html -- -M HEAD~1', 'print to file')
-  .version(function() {
```

Comment:
Why are you removing this?

Reply:
Because I have no idea what I'm doing. :).  This was my first foray into anything NPM related, actually.

I didn't touch that directly.  It must have been fallout from what I've done.

I'll try to find moment to submit a fixed request.

An option to suppress the credits would be handy, I suppose, but the credits being in the footer aren't a problem, either, and, really, thanks!

Reply:
I think I know what is the problem. You did the pr to the wrong branch. Open a new one to master.